### PR TITLE
Don’t use block with RestClient

### PR DESCRIPTION
**Pull request by bfcoder**
Allow RestClient to throw errors when the api it connects to returns 4XX and 5XX codes.

### CODE REVIEW

File: app/services/scorm_engine_service.rb [hunk 1]
```diff
@@ -60,18 +60,16 @@ def import_course(file, course_id)
...
         password: @api_password,
         payload: {
           file: UploadIO.new(zip, "zip/zip", File.basename(file)),
         },
-      ) do |response|
-        raise response if response.code == 500
-        JSON.parse(response.body)["result"]
-      end
+      )
+      JSON.parse(response.body)["result"]
```
Comment (jbasdf) [inline app/services/scorm_engine_service.rb]: If RestClient::Request.execute returns an error (a 503 for example) will an exception be thrown or will it attempt to do a JSON.parse on an invalid response.body?

Comment (bfcoder) [reply app/services/scorm_engine_service.rb]: An exception will be thrown. It won't get to this parse code.

Comment (bfcoder) [reply app/services/scorm_engine_service.rb]: I've tested this and RestClient will throw a 500 for a 404 and a 503 from the api it connects to. So we will never have to worry about a bad JSON.parse.

Comment (jbasdf) [reply app/services/scorm_engine_service.rb]: Perfect

**Pull request merged**